sdk/template-dpg/Azure.ServiceTemplate.Template/src/Azure.ServiceTemplate.Template.csproj

```diff
@@ -29,10 +26,10 @@
     <Compile Include="$(AzureCoreSharedSources)OperationInternalBase.cs" LinkBase="Shared" />
     <Compile Include="$(AzureCoreSharedSources)OperationInternalOfT.cs" LinkBase="Shared" />
     <Compile Include="$(AzureCoreSharedSources)AzureResourceProviderNamespaceAttribute.cs" LinkBase="Shared" />
-    <Compile Include="$(AzureCoreSharedSources)Argument.cs" LinkBase="Shared" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\..\core\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj" />
```

Comment:
These should be package references not project references.

Reply:
If use the package references, sometime the Azure.Core.Experimental is not match the autorest.csharp, and may fail to build the generated code.  After the Azure.Core, Azure.Core.Experimental, and autorest.csharp are stable, we will change back to use package reference

Reply:
If someone releases a package with a project reference, though, nuget will turn it into a package reference to a package that does not exist.  Any customer code that references the released package will fail because the dependent package will not be found.

Reply:
updated to use package reference by now